3/1-Introduction.md

```diff
@@ -0,0 +1,215 @@
+
+# OpenCannabis Specification
+
+### Status of this Memo
+
+This memo defines a protocol and model for the legal cannabis community: _OpenCannabis, version 1.0_. Distribution of
+this memo is unlimited.
```

Comment:
here's the introduction to the spec - essentially, section 1. everything is numbered in directory `3/`, because that's `OCS`'s spec code/number.

inside that directory, chapters or parts are designated by `N-`, as in `1-Introduction.md`, and `2-Datamodel.md`, and so on.